README.md

```diff
@@ -37,48 +37,50 @@ on the host system, the minimum build requirements:
 - bc
 
 ## building
-set `DEFCONFIG` to one of the [supported boards](#supported-boards), `DEFCONFIG=rpi3` for example.
-first, configure
+set `DEFCONFIG` to one of the [supported boards](#supported-boards),
 
 ```sh
-. env.sh
-make --directory=$(pwd)/buildroot O=$(pwd)/${DEFCONFIG}_output ${DEFCONFIG}_defconfig
+$ export DEFCONFIG=rpi4
+```
+
+for example.  first, configure
+
+```sh
+$ ./build.sh ${DEFCONFIG}_defconfig
```

Comment:
enhancement: it would be nice if we didn't have to do this ourselves and could just pass `defconfig`

Reply:
eh



meh